index.html

```diff
@@ -134,7 +134,28 @@ <h1>About</h1>
             </li>
           </ul>
       </section>
+      <section class="contact">
+        <h1>Get Started</h1>
```

Comment:
[OPTIONAL]
- [ ] It is advisable for best practices and HTML standards to make use of only one `h1` tag in the whole of your application check out this [link](https://www.reliablesoft.net/h1-tag/) for more information. i kindly suggest you make use of another heading tag such as `h2, h3, h4` etc:+1: